internal/testing/ci/smoke/smoke_test.go

```diff
@@ -77,11 +77,15 @@ func TestSmoke(t *testing.T) {
 			},
 		},
 		{
-			name:        "kubernetes success",
-			driver:      "kubernetes",
-			circleHost:  cli.Tests.Kubernetes.CircleHost,
-			circleToken: cli.Tests.Kubernetes.CircleToken,
-			namespace:   cli.Tests.Kubernetes.RunnerNamespace,
+			name:   "kubernetes success",
+			driver: "kubernetes",
+			//circleHost:  cli.Tests.Kubernetes.CircleHost,
+			//circleToken: cli.Tests.Kubernetes.CircleToken,
+			//namespace:   cli.Tests.Kubernetes.RunnerNamespace,
+			// TODO: Temporarily run the Kubernetes smoke tests on cloud until we complete https://circleci.atlassian.net/browse/ONPREM-2237
```

Comment:
Testing: https://app.circleci.com/pipelines/github/circleci/runner-smoke-tests/19730/workflows/2a5f6226-6aa9-45c2-a5ab-600bd9f104f9

Note that our machine runner tests run in cloud, so swapping some values to those works